src/apiCaller.js

```diff
@@ -1,13 +1,26 @@
 import PageData from './components/pageData'
+import path from 'path'
 
 const apiCaller = (opts) => {
   return (files, metalsmith, done) => {
+    const postQuery = (opts.url && opts.url.postBuild) || false
     new PageData({
       opts,
       files
     }).then((data) => {
-      files = data
-      done()
+      const newDone = data => {
+        files = data
+        done()
+      }
+      if (postQuery) {
+        const pathToFunc = path.join(metalsmith._directory, postQuery)
+        const func = require(pathToFunc)
+        if (typeof func !== 'function') return newDone(data)
+        func(data).then(newDone).catch(function (err) {
+          console.log(err)
```

Comment:
Do we need this?

Reply:
I see from the PR associated with this change, this was put into support a post query- so I think it is required to be kept.